torchci/pages/metrics.tsx

```diff
@@ -383,12 +396,25 @@ export default function Page() {
               badThreshold={(value) => value > 60 * 60 * 6} // 6 hours
             />
             <ScalarPanel
-              title={"Trunk workflow duration avg"}
-              queryName={"workflow_duration_avg"}
+              title={"Trunk workflow duration p50"}
+              queryName={"workflow_duration_percentile"}
+              metricName={"duration_sec"}
+              valueRenderer={(value) => durationDisplay(value)}
+              queryParams={[
+                { name: "name", type: "string", value: "trunk" },
+                { name: "percentile", type: "float", value: 0.50 },
+                ...timeParams,
+              ]}
+              badThreshold={(value) => value > 60 * 60 * 3} // 3 hours
+            />
+            <ScalarPanel
+              title={"Trunk workflow duration p90"}
```

Comment:
nit: maybe call it "p90 Trunk workflow duration" to be consistent with the other names.